jsx/Form.js

```diff
@@ -1104,19 +1113,23 @@ DateElement.propTypes = {
   dateFormat: PropTypes.string,
   disabled: PropTypes.bool,
   required: PropTypes.bool,
+  hasError: PropTypes.bool,
+  errorMessage: PropTypes.string,
   onUserInput: PropTypes.func,
 };
 
 DateElement.defaultProps = {
   name: '',
   label: '',
-  value: '',
+  value: undefined,
```

Comment:
@laemtl should value be undefined? Line 1054 still checks `this.props.value === ''` if empty string. I'm wondering if null is more appropriate instead of undefined if empty string isn't preferred for some reason?

Reply:
If value is set to '', an error is detected on page load (initial value enter the error test case). value can be null instead, but other element (select for example) use undefined as their initial value. Any reason why undefined should be avoided?

Reply:
I've not witnessed undefined from an input tag before and so I was just curious. I get an empty string when the variable has not been set for <input> if I make one and try to console.log the value. I'm guessing date works differently.